### PR TITLE
Move CI to Github Actions

### PR DESCRIPTION
**Pull request by gavin-stackrox**
This PR moves this repos CI to Github Actions. It is pretty much a direct copy i.e. this is not new code.
I did not move the auto-creation of PRs in dependent repos as that (IMO) needs to be reconsidered to use action features.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,84 +2,80 @@ ifeq ($(STACKROX_CENTOS_TAG),)
...
-stackrox-test-cci-image:
-	$(DOCKER) build \
-	    -t stackrox/apollo-ci:$(STACKROX_TEST_CCI_TAG) \
-	    -t quay.io/$(QUAY_REPO)/apollo-ci:$(STACKROX_TEST_CCI_TAG) \
-		--build-arg BASE_TAG=$(STACKROX_TEST_TAG) \
-		-f images/circleci.Dockerfile \
-		images/
-
 .PHONY: test-cci-export
 test-cci-export:
```
Comment (RTann) [inline Makefile:44]: is this one still needed?

File: images/test.cci-export.Dockerfile [hunk 2]
```diff
@@ -1,9 +1,8 @@
 ARG BASE_TAG
```
Comment (RTann) [inline images/test.cci-export.Dockerfile:1]: Is this file still needed? Or just renamed?

File: images/test/bats/cci-export.bats [hunk 3]
```diff
@@ -4,20 +4,19 @@
 # make rocksdb-image
```
Comment (RTann) [inline images/test/bats/cci-export.bats:4]: should we rename this file?

File: Makefile (hunk 1 above)
Comment (gavin-stackrox) [reply Makefile:44]: yes. cci-export is still used in OSCI and still tested here.

File: images/test.cci-export.Dockerfile (hunk 2 above)
Comment (gavin-stackrox) [reply images/test.cci-export.Dockerfile:1]: it is still called cci-export in dependent repos. so yeah. still needed.

File: images/test/bats/cci-export.bats (hunk 3 above)
Comment (gavin-stackrox) [reply images/test/bats/cci-export.bats:4]: no :)

File: Makefile [hunk 4]
```diff
@@ -2,84 +2,80 @@ ifeq ($(STACKROX_CENTOS_TAG),)
...
 ifeq ($(ROCKSDB_TAG),)
-ROCKSDB_TAG=$(shell .circleci/get_tag.sh "rocksdb" "$(STACKROX_CENTOS_TAG)")
+ROCKSDB_TAG=$(shell scripts/get_tag.sh "rocksdb" "$(STACKROX_CENTOS_TAG)")
 endif
 ifeq ($(DOCKER),)
 DOCKER=docker
 endif
 QUAY_REPO=rhacs-eng
 
+ROCKSDB_IMAGE=
```
Comment (RTann) [inline Makefile:12]: what is this for?

Comment (gavin-stackrox) [reply Makefile]: :shame:

File: Makefile [hunk 5]
```diff
@@ -2,84 +2,80 @@ ifeq ($(STACKROX_CENTOS_TAG),)
...
-		-it \
+		--rm \
 		test-cci-export
 
-.PHONY: collector-test-image
-collector-test-image:
+.PHONY: collector-image
+collector-image:
 	$(DOCKER) build \
+	    -t quay.io/$(QUAY_REPO)/apollo-ci:$(shell scripts/get_tag.sh "collector") \
```
[the remark this answers is not in the document]
Comment (gavin-stackrox) [reply Makefile]: substituted tabs for space (as it should be in Makefiles).
